clients/unshaded/src/main/java/tachyon/client/RawTableMasterClient.java

```diff
@@ -66,6 +67,8 @@ protected void afterConnect() {
   /**
    * Creates a raw table. A table is a directory with sub-directories representing columns.
    *
+   * This method does not use retries because it is not idempotent.
```

Comment:
This seems a temporary solution. I think we still need retry, but we can add the state cleanup when an exception is thrown.